io_mesh_w3d/export_utils.py

```diff
@@ -90,12 +84,16 @@ def retrieve_meshes(context, hierarchy, rig, container_name):
             vert_materials=[],
             textures=[],
             material_passes=[],
-            shader_materials=[])
+            shader_materials=[],
+            multi_bone_skinned=False)
 
         header = mesh_struct.header
         header.mesh_name = mesh_object.name
         header.container_name = container_name
 
+        if mesh_object.userText != '':
+            mesh_struct.user_text = mesh_object.userText
```

Comment:
Why is that check required?

Reply:
since it is None by default, and this way the userText chunk is only written if it has a meaningfull userText, otherwise every exported mesh would have that chunk even if it contains '' all the time

Reply:
But when it's `None` the condition is true?

Reply:
I think `if mesh_object.userText:` would be more correct.

Reply:
i mean: only set mesh_struct.user_text if mesh_object.userText is not ''. mesh_struct.user_text is None by default which is checked on writing the mesh struct if a corresponding chunk should be written

Reply:
Yes, but this check doesn't exclude strings which are None

Reply:
thats true, but on import the text is set to '' but dont know what the value would be on newly created meshes, ill have a look

Reply:
changed that, since the userText of newly created objects is '' by default